core/tests/testbed/customize/sitecustomize.py

```diff
@@ -1,3 +1,9 @@
-import coverage
+import os
 
-cov = coverage.process_startup()
+# Only set up the app processes for coverage *if* coverage is running.
+# Otherwise, just importing coverage will force running coverage.
+# The COVERAGE_RUN environment variable is set by coverage.py itself.
+if os.environ.get("COVERAGE_RUN"):
+    import coverage
+
+    coverage.process_startup()
```

Comment:
FTR, this was driving me absolutely insane until I discovered this. I was pulling my hair out trying to figure out how to disable coverage 🤣 very sneaky